contracts/ToucanRegenBridge.sol

```diff
@@ -38,6 +38,8 @@ contract ToucanRegenBridge is Ownable, Pausable {
     event Bridge(address sender, string recipient, address tco2, uint256 amount);
     /// @notice emited when we bridge tokens back from Regen Ledger and issue on TCO2 contract
     event Issue(string sender, address recipient, address tco2, uint256 amount);
+    /// @notice emited when the token issuer is updated
+    event TokenIssuer(address oldIssuer, address newIssuer);
```

Comment:
i feel `TokenIssuer` is not really a good event name. i'd personally pick something like `TokenIssuerChanged`

Reply:
Updated to `TokenIssuerUpdated` to stay aligned with terminology we use elsewhere.